docker/Makefile

```diff
@@ -0,0 +1,35 @@
+run: compile tmux
+
+M_UID=$(shell id -u)
+M_GID=$(shell id -g)
+
+
+gen:
+	export MIMIKER_UID=$(M_UID) MIMIKER_GID=$(M_GID) && \
+	envsubst < Dockerfile.tpl > Dockerfile
+
+build: gen
+	docker build --no-cache . -t mimiker-dev:latest
+
+up:
+	touch /tmp/.docker.xauth
+	touch ~/.tmux.conf
+	touch ~/.gitconfig
+	xauth nlist :0 | sed -e 's/^..../ffff/' | xauth -f /tmp/.docker.xauth nmerge -
+	docker-compose up -d
+
+down:
+	docker-compose down
+
+compile: up
+	docker-compose exec mimiker make
+
+tmux: up
+	docker-compose exec mimiker tmux
+
+clean:
+	docker container prune --filter "until=24h"
+	docker image prune
+	rm Dockerfile
```

Comment:
```suggestion
	rm -f Dockerfile
```

Reply:
@jpszczolowski I don't think that there is a need for `-f` flag here, are there any problems with permissions during deleting that file on your machine?

Reply:
I think that one usually adds `-f` to `rm` inside `Makefile`s because of `rm: cannot remove 'Dockerfile': No such file or directory` error if the file does not exist (e.g. double `make clean`).

Reply:
`-f` means `ignore nonexistent files and arguments`, there is no problem with permissions

Reply:
Ah, of course. Good catch

Reply:
FYI, `$(RM)` means `rm -f` inside a `Makefile`
https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html